pkg/kapp/app/app_filter.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package app
+
+import "time"
+
+type AppFilter struct {
+	CreatedAtBeforeTime *time.Time
+	CreatedAtAfterTime  *time.Time
+
+	Labels []string
```

Comment:
what uses this?

Reply:
Getting used to store labels got from `--filter-labels` flag